src/dtos/comments.dto.ts

```diff
@@ -0,0 +1,22 @@
+import { IsDate, IsNumber, IsString } from 'class-validator';
+
+
+export class CreateCommentDto {
+    @IsNumber()
+    public post_id: number;
+
+    @IsNumber()
+    public user_id: number;
+
+    @IsString()
+    public media: string;
+
+    @IsString()
+    public context: string;
+
+    @IsDate()
```

Comment:
We don't need created_at and updated_at in DTO.
DTOs define the incoming request object
Created_at and updated_at are handled by typeorm and not the client

Reply:
Is there a need to use Class Validator if we're using typescript anyway?